app/controllers/chargeback_controller.rb

```diff
@@ -227,12 +227,24 @@ def cb_assign_field_changed
     cb_assign_get_form_vars
     render :update do |page|
       page << javascript_prologue
-      changed = (@edit[:new] != @edit[:current])
+      changed = selections_changed?
```

Comment:
Wouldn't

```ruby
changed = @edit[:new].except(key1, key2, ...) != @edit[:current].except(key1, key2, ...)
```

suffice here?

Reply:
For me this does not work somehow... 
```
[28] pry(#<#<Class:0x00562213097370>>)> except
=> ["cbshow_typ", "cbtag_cat", "cblabel_key"]
[29] pry(#<#<Class:0x00562213097370>>)> @edit[:new]
=> {:cbshow_typ=>"storage-tags", :cbtag_cat=>"2", "storage-tags__2"=>"3"}
[30] pry(#<#<Class:0x00562213097370>>)> @edit[:new].except(*except)
=> {:cbshow_typ=>"storage-tags", :cbtag_cat=>"2", "storage-tags__2"=>"3"}
[31] pry(#<#<Class:0x00562213097370>>)> @edit[:new].except(:cbshow_typ)
=> {:cbtag_cat=>"2", "storage-tags__2"=>"3"}
```

Reply:
`"cbshow_typ" != :cbshow_typ`

Reply:
Yes, my bad, `%i`

Reply:
Ahh, thank you, guys! :+1: :heart:

Reply:
> Yes, my bad, %i

Nope, my comment was a reply to an earlier comment, which got deleted in the meantime.